documentation/jetty-documentation/src/main/asciidoc/operations-guide/begin/deploy.adoc

```diff
@@ -46,46 +46,46 @@ mywebapp.war
 <5> `WEB-INF/web.xml` is the web application deployment descriptor, which defines the components and the configuration of your web application.
 ====
 
-To deploy a standard web application, you need to enable the xref:og-module-deploy[`deploy` module]:
+To deploy a standard web application, you need to enable the xref:og-module-eeN-deploy[`{ee-current}-deploy` module].
+
+NOTE: The following examples assume you're deploying a Jakarta {ee-current-caps} app; for other versions of EE, make sure to activate the corresponding `{ee-all}-deploy` module.
 
 ----
-$ java -jar $JETTY_HOME/start.jar --add-module=deploy
+$ java -jar $JETTY_HOME/start.jar --add-module={ee-current}-deploy
 ----
 
 [source,options=nowrap]
 ----
-include::jetty[setupArgs="--add-module=http",args="--add-module=deploy"]
+include::jetty[setupArgs="--add-module=http",args="--add-module={ee-current}-deploy"]
 ----
 
-The `deploy` module creates `$JETTY_BASE/webapps`, which is the directory where Jetty looks for any `+*.war+` files or web application directories to deploy.
+The `{ee-current}-deploy` module creates `$JETTY_BASE/webapps`, which is the directory where Jetty looks for any `+*.war+` files or web application directories to deploy.
 
-[NOTE]
-====
-Activating Jetty's `deploy` module enables web application deployment.
+Activating one of Jetty's `{ee-all}-deploy` modules enables web application deployment.
```

Comment:
@sbordet: I've been using the words "enable", "activate", and "add" pretty much interchangeably when referring to modules. Is one of these terms preferred? If not, it may make sense to standardize the terminology.

Another way to think about it: are any of the following sentences more conceptually accurate? Or are they all about the same?
* When I **add** a module, it **activates** one of Jetty's features.
* When I **activate** a module, Jetty **enables support for** some feature.
* When I **enable** a module, it **adds support for** some feature in Jetty.

Reply:
Good question, I would not use "activate" and stick with "enable" and "add" interchangeably.

"When you **add** a module, it **enables** support for feature X" 

or similarly:

"When you **enable** a module, it **adds** support for feature X".

Reply:
Sounds good. I'll use this convention moving forward, and will fix existing usages as I come across them.